cmd/xfn/spark/spark.go

```diff
@@ -69,6 +69,9 @@ const ociRuntimeRoot = "runtime"
 // the RunFunctionRequest.
 const defaultTimeout = 25 * time.Second
 
+// The maximum size of stdout/stderr to avoid OOM
+const maxStdioBytes = 100 << 20 // 100 MB
```

Comment:
Should this be a CLI flag? I think that would let us avoid defining it twice - we could define one in `container_linux.go` and pass it as an argument to `xfn spark`.

Reply:
See latest commit (https://github.com/crossplane/crossplane/pull/4217/commits/0be527693644727e2eda44d99ecd0e7cd504b4e2)